App.js

```diff
@@ -26,7 +26,7 @@ SplashScreen.preventAutoHideAsync();
 
 const App = () => {
   const appState = useRef(AppState.currentState);
-  const [isFontLoaded, setIsFontLoaded] = useState(false);
+  const [isFontLoaded] = Font.useFonts(fontConfig);
```

Comment:
wow, is this new since latest expo sdk updates?

Reply:
As far as I can see, this is a new hook that comes with SDK 50.